cypress/test/scripts/tools/extract-tools-github.cy.js

```diff
@@ -0,0 +1,199 @@
+import { getData } from "../../../../scripts/tools/extract-tools-github";
+describe('Get Data', () => {
+    // eslint-disable-next-line cypress/no-async-tests
```

Comment:
Why did we add this comment?

Reply:
cypress eslint does n't  allow async and await , thus to disable eslint this was added

Reply:
You can ignore the eslint warnings in the files.